open_buildings/cli.py

```diff
@@ -59,21 +61,21 @@ def get_buildings(geojson_input, dst, source, country_iso, silent, overwrite, ve
     not use country_iso. In future versions of this tool we hope to eliminate the need to hint with the country_iso.
     """
     # map source of google and overture to values for data_path and hive
-    data_path = None
-    hive_partitioning = False
     # case insensitive matching
-    if source.lower() == "google":
-        data_path = "s3://us-west-2.opendata.source.coop/google-research-open-buildings/geoparquet-by-country/*/*.parquet"
-        hive_partitioning = True
-    elif source.lower() == "overture":
-        data_path = "s3://us-west-2.opendata.source.coop/cholmes/overture/geoparquet-country-quad-hive/*/*.parquet"
-        hive_partitioning = True
+    match source.lower():
```

Comment:
i realise match case was only introduced in python 3.10 - will revert to if/else to support older versions

Reply:
Ok, sounds great. As I mentioned in the other comment I don't know the tradeoffs in python well enough to reason about what version to target, but I generally lean towards making it easier for users as long as it's not onerous for devs.

Reply:
yeah, i think it's better to support older versions if it's not too much hassle :)